### PR TITLE
Release 3.0.1

### PR DESCRIPTION
**Pull request by machacekondra**
Release 3.0.1

### CODE REVIEW

**Comment by alinabuzachis** [comment]
recheck

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,6 +4,14 @@ vmware.vmware_rest Release Notes
 
 .. contents:: Topics
 
+v3.0.1
+===========
+
```
Comment (GomathiselviS) [inline CHANGELOG.rst:9]: Can you add release_summary? (Refer: https://github.com/ansible-collections/vmware.vmware_rest/pull/481/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43cL29)

File: CHANGELOG.rst [hunk 2]
```diff
@@ -26,11 +34,6 @@ Breaking Changes / Porting Guide
 v2.3.1
 ======
 
-Release Summary
```
Comment (GomathiselviS) [inline CHANGELOG.rst:29]: Please add this back

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -671,3 +671,13 @@ releases:
     - start_gha_4_sanity_tests.yaml
     - use_7.0.3_spec.yaml
     release_date: '2023-11-27'
+  3.0.1:
+    changes:
+      bugfixes:
+      - content_library_item_info - fixed error with unsupported property
+      - lookup plugins - Refactor to use native options configuration via doc_fragment,
+        which ensures that vcenter_validate_certs=false is honored (https://github.com/ansible-collections/vmware.vmware_rest/issues/425).
+    fragments:
```
Comment (GomathiselviS) [inline changelogs/changelog.yaml:680]: Add release summary

**Review by GomathiselviS — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (machacekondra) [reply CHANGELOG.rst:29]: Sure! Not sure how come it was removed.

**Comment by machacekondra** [comment]
Closing, as wrong branch was used. Will repost to stable-3.

**Pull request closed without merging**